src/main/flight/mixer.c

```diff
@@ -653,6 +654,14 @@ FAST_CODE_NOINLINE void mixTable(timeUs_t currentTimeUs)
     }
 #endif
 
+#ifdef USE_ALTHOLD_MODE
+    if (ARMING_FLAG(ARMED) && !failsafeIsActive()) {
+        float altHoldThrottle = getAltHoldThrottle();
+        float altHoldThrottleFactor = getAltHoldThrottleFactor(throttle);
+        throttle = throttle * (1.0f - altHoldThrottleFactor) + altHoldThrottle * altHoldThrottleFactor;
```

Comment:
```suggestion
        throttle += (altHoldThrottle - throttle)  * altHoldThrottleFactor;
```
This rearrangement is used quite commonly and is slightly faster. But alpha :  (1 - alpha) version is slightly easier to read